library/core/class.session.php

```diff
@@ -29,15 +29,15 @@ class Gdn_Session {
     /** @var object Attributes of the current user. */
     protected $_Attributes;
 
-    /** @var object Permissions of the current user. */
-    protected $_Permissions;
-
     /** @var object Preferences of the current user. */
     protected $_Preferences;
 
     /** @var object The current user's transient key. */
     protected $_TransientKey;
 
+    /** @var  Vanilla\Permissions */
+    private $permissions;
```

Comment:
Gonna hold off on this for now, since `Gdn_Session` already has a `getPermissions` and a `setPermission`.  We'll likely deprecate/replace these methods in the future.